claims/src/claims.rs

```diff
@@ -0,0 +1,163 @@
+//
+// Copyright 2023 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+//! This module contains structs for specifying claims about software artifacts.
+
+extern crate alloc;
+
+use crate::intoto::{DigestSet, Predicate, Statement, STATEMENT_INTOTO_V01};
+use alloc::{string::String, vec::Vec};
+use anyhow::Context;
+use serde::{Deserialize, Serialize};
+use time::OffsetDateTime;
+
+/// URI representing the PredicateType of a V1 Claim. To be used in in-toto statements.
+pub const CLAIM_V1: &str = "https://github.com/project-oak/transparent-release/claim/v1";
+
+/// ClaimType for Endorsements. This is expected to be used together with `ClaimV1` as the predicate
+/// type in an in-toto statement.
+pub const ENDORSEMENT_V2: &str =
+    "https://github.com/project-oak/transparent-release/endorsement/v2";
+
+/// Trait representing a claim specification.
+pub trait ClaimSpec {}
```

Comment:
Not sure I understand the point of this empty trait, could you add a comment? If it's a marker trait, please make that explicit. The fact that it's public though does not really make it useful, since anyone can trivially implement it on any type.

Reply:
Good point. Removed it.